test/run-stp-tests.sh

```diff
@@ -490,7 +488,7 @@ fi
 cat >$BASE/cupsd.conf <<EOF
 StrictConformance Yes
 Browsing Off
-Listen localhost:$port
+Listen 127.0.0.1:$port
```

Comment:
Again, libcups maps localhost to both IPv4 and IPv6 addresses when the local platform supports IPv6 (as all modern Linux kernels do...) Not going to limit the test to IPv4...